article/sections/_4_graph_construction.tex

```diff
@@ -0,0 +1,109 @@
+How do the actions of a decision maker $i$ at time $t$ partially cause the future context or outcomes of a decision maker $j$?
+We should add explicit nodes to our graph, subscripted or denoted by time, to show the cross-time causal relationships in our system.
+
+At this point, we have added to our causal graph all the outcome, explanatory, effect modifying, mediating, confounding, and time-indexed variables that we believe are relevant for our problem.
+However, they are all disconnected nodes, i.e., singletons in the graph.
+We now focus on pruning nodes from this graph, before drawing our final hypothesized connections.
+In particular, we focus on pruning ``post-outcome'' variables that are not part of the causal graph for future time periods or other observations.
+The reason for this is that conditioning on such post-outcome variables would bias our causal effect estimates.
+
+To remove the problematic variables, we iterate through each of the non-outcome variables in our graph, and we assess whether each variable is actually a result of the outcome (perhaps in combination with other variables in our graph).
+These post-outcome variables temporally follow the outcome variable(s) but do not cause variables in the causal graph for other observations.
+We remove all such post-outcome variables from our graph.
+
+Now is a good time to step back and consider what other researchers have thought about our problem.
+Specifically, we should conduct a literature review to see how other researchers have conceptualized the topic that we are working on.
+Have they included variables that we have not?
+Were those variables related our outcomes of interest?
+If so, should we add these variables to our causal graph? How should these variables enter our graph?
+Do the included variables of other researchers suggest the existence of confounders in their work that we should include in our graph?
+Have other researchers ascribed differing roles to our graph's current variables than we have?
+For example, have other researchers judged a variable to be a confounder, when we solely thought of the variable as an effect modifier?
+As we answer these questions, we should critically examine the evidence for these alternative decisions to see if we should also reconsider how we're judging our variables.
+
+Finally, we need to connect the variables in our graph.
+\begin{enumerate}
+   \item Draw direct arrows from our explanatory variables, confounders, and effect modifiers to the outcomes.
+   \item Draw arrows from the explanatory variables to the mediators, and then draw arrows from the mediators to the outcomes.
+   \item Draw arrows from the confounders to the explanatory variables and mediators that they may cause.
+   \item Draw arrows from the variables in time $t$ to the variables that they cause in time $t+1$.
+\end{enumerate}
```

Comment:
Update this section once we have a finalized set of variables to be considered for inclusion

Reply:
See edits in #165.